tests/unit/store/adapter-interop-test.js

```diff
@@ -113,6 +113,33 @@ test('Calling Store#findRecord multiple times coalesces the calls into a adapter
   });
 });
 
+test('Coalesced Store#findRecord requests retain the `include` adapter option in the snapshots passed to adapter#findMany', function(assert) {
+  assert.expect(2);
+
+  const Adapter = TestAdapter.extend({
+    findMany(store, type, ids, snapshots) {
+      snapshots.forEach(snapshot => {
+        assert.equal(snapshot.include, 'someResource', 'Snapshots retain the `include` adapter option');
+      });
+      return Ember.RSVP.resolve({ data: [{ id: 1, type: 'test' }, { id: 2, type: 'test' }] });
+    },
+    coalesceFindRequests: true
+  });
+
+  const Type = DS.Model.extend();
+  let store = createStore({
+    adapter: Adapter,
+    test: Type
+  });
+
+  return run(() => {
+    return Ember.RSVP.all([
+      store.findRecord('test', 1, { include: 'someResource' }),
+      store.findRecord('test', 2, { include: 'someResource' })
```

Comment:
Next step would be requesting different `include` strings and fixing [`groupRecordsForFindMany`](https://www.emberjs.com/api/ember-data/3.0/classes/DS.JSONAPIAdapter/methods/groupRecordsForFindMany?anchor=groupRecordsForFindMany) to take that into account, I guess?

Reply:
Also, I should probably already request different `include`s here and assert, that these _different_ `include`s are correctly passed to `findMany`.

Reply:
I'll give this a review this evening :)

Reply:
I agree with your proposed test enhancements :)

I think we pass on the grouping for now though.